koku/api/provider/provider_manager.py

```diff
@@ -67,15 +66,11 @@ def get_name(self):
         """Get the name of the provider."""
         return self.model.name
 
-    def get_infrastructure_name(self, tenant):
+    def get_infrastructure_name(self):
         """Get the name of the infrastructure that the provider is running on."""
-        provider_accessor = ProviderAccessor(self.model.type)
-        try:
-            infra_type = provider_accessor.infrastructure_type(self._uuid, tenant)
-        except ProviderAccessorError as error:
-            LOG.error('Unable to determine infrastructure type. Reason: %s', str(error))
-            infra_type = 'Unknown-Error'
-        return infra_type
+        if self.model.infrastructure and self.model.infrastructure.infrastructure_type:
+            return self.model.infrastructure.infrastructure_type
+        return 'Unknown'
```

Comment:
This is the core of the PR.

Reply:
Is the infrastructure_type the name that the provider is running on?

Reply:
Yes